packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -105,12 +106,21 @@ export class AtomicBreadbox implements InitializableComponent {
     this.updateShowMoreValue(hiddenBreadcrumbs);
   }
 
+  private updateShowLessDisplay() {
+    this.show(this.showLess);
+    if (this.showLess.offsetTop === 0) {
+      this.hide(this.showLess);
+    }
+  }
```

Comment:
If the goal is to hide some breadcrumb values, but only when they don't fit in one line, then perhaps it would be better to update the `isOverflowing` getter instead to match this condition: "the component is overflowing when an element has offsetTop != 0".

Reply:
Different path, this is when we show all values (no overflow) and we check if the "show less" button appears on the first row or not, if it appears on the first row it's not necessary